slack-autoarchive.py

```diff
@@ -13,18 +13,22 @@
 ADMIN_CHANNEL      = os.getenv('ADMIN_CHANNEL')
 AUDIT_LOG          = 'audit.log'
 DAYS_INACTIVE      = 60
+MIN_MEMBERS        = 5
```

Comment:
Sure, sounds fine.  I only chose 5 due to the README. (However, so far 5 has proved to work well for my 1700 channel Slack instance!)